apps/www/app/routes/_index.tsx

```diff
@@ -18,108 +25,129 @@ export const meta: MetaFunction = () => {
-            className="pl-2 flex items-center">
-            <Text.Small className="text-center">Docs</Text.Small>
-          </a>
-        </div>
-      </header>
-      <div className="grid grid-cols-1 md:grid-cols-2 h-full w-full">
-        <div className="flex flex-col justify-center items-start p-10 gap-8">
-          <div>
-            <h1 className="text-foreground text-5xl md:text-7xl font-bold">
-              Developer Focused Server-Driven Experiences
-            </h1>
-            <Text.P className="font-thin leading-normal text-xs md:text-sm">
-              Securely render externally hosted components in React Native.
-              Developer or creator this tool is for you, remotely host
-              components or whole screen experiences.
-            </Text.P>
-          </div>
-          <div className="flex flex-row gap-4">
-            <Link to="/chunks">
-              <Button>
-                <Text.P>Get Started</Text.P>
-                <CaretRightIcon className="size-5" />
-              </Button>
-            </Link>
-            <Link to="https://crherman7.github.io/rechunk/">
-              <Button>
-                <Text.P>Docs</Text.P>
-                <ExternalLinkIcon className="size-4 ml-2" />
-              </Button>
-            </Link>
-          </div>
-        </div>
-        <div className="hidden md:block">
-          <div className="fixed mt-12 mr-10 flex flex-col items-end">
-            <div className="bg-green-500 px-2">
-              <p className="text-[8px] text-white">chunk_001</p>
-            </div>
-            <img
-              src="/feed.png"
-              className="h-[250px] object-contain border-[1px] border-green-500 ml-6"
-              alt="feed"
-            />
+    <div className="min-h-screen flex flex-col relative overflow-hidden">
+      {/* Background Grid */}
+      <GridPattern
+        width={30}
+        height={30}
+        x={-1}
+        y={-1}
+        strokeDasharray="4 2"
+        className="[mask-image:radial-gradient(750px_circle_at_center,white,transparent)] -z-10"
+      />
+
+      {/* Header Section */}
+      <Header />
+
+      {/* Main Content */}
+      <main className="flex flex-col items-center px-6 py-6 mt-6">
+        <ShinyDiv id="myElement" className="mb-12 rounded-full opacity-0">
+          <div className="flex flex-row justify-center items-center gap-3">
+            <p>🎉</p>
+            <div className="h-4 w-[0.1px] bg-gray-300" />
+            <Text.Small className="text-gray-500 font-light text-xs">
+              Beta Coming Soon
+            </Text.Small>
           </div>
-          <img
-            src="/arrow_11.svg"
-            className="size-20 rotate-[270deg] fixed mt-[340px] ml-28"
-            alt="arrow"
+        </ShinyDiv>
+        <Text.H1 id="myElement" className="text-center opacity-0">
+          Launch without limits.
+        </Text.H1>
+        <Text.P
+          id="myElement"
```

Comment:
🤔 what do you think about renaming this?

Reply:
good call!

Reply:
updated :)